### ISSUE TITLE
Station search with nordic characters malfunction

### ISSUE THREAD

**Issue by abkro** [smurfy/fahrplan#189]
I am using Fahrplan with the Norwegian API. 

Many place names contain one or more of the characters å, æ, ø. App behavior is inconsistent in these cases, but it looks as though it is unable to find the names starting with one of these characters. 

In addition, when the character is not at the beginning of the name, some names are found, but others not. 

Examples:

Åros (several bus stops starting with this name). Does not turn up.

Oksenøyveien. Bus stop. Does not turn up.

Røyken. Works fine. Several results. Both train and bus.

I can supply more examples upon request.

**Comment by abkro** [comment]
Submitted from my Ubuntu device. Unable to find how to assign tags to this issue, i.e. [Ubuntu]

**Comment by abkro** [comment]
Submitted from my Ubuntu device. Unable to find how to assign tags to this issue, i.e. [Ubuntu]

**Comment by leppa** [comment]
I checked on BlackBerry 10 and I can find mentioned stops. So it might be Ubuntu specific issue.

> Unable to find how to assign tags to this issue, i.e. [Ubuntu]

Do you mean labels that are displayed on the right? Only Collaborators can do that, AFAIK.

**Comment by mzanetti** [comment]
http://i.imgur.com/HNAWHxt.png

Seems to work for me. Note that I have searched for "Aros", not "Åros" as I don't have that character and I don't seem to have an option to set the keyboard layout to norwegian on my phone. Does this only happen when you use those special characters for searching?

Another thing to check is that you're not confusing it with bug #188.

**Comment by abkro** [comment]
Yes, I checked and I also get the stations Åros ... when typing with an 'A' instead of 'Å'. And yes again, it only happens when I use the mentioned special characters. (Incidentally, you get these with any of the nordic keyboard layouts, i.e. Swedish, Danish, Norwegian). Finally, I am not confusing this with bug #188.

**Comment by abkro** [comment]
Hello again,

I have now joined the translator team and finished translating Fahrplan to Norwegian. The bug with the Nordic characters (Å, Æ, Ø), however, has not been solved. Would someone please look into this issue?

A and Å are different characters in the alphabet, with different pronunciations. Some users may figure out they can use A instead of Å and so on, but many will probably not, and all will find the issue annoying. Switching to using the other similar letters, therefore, is not a serious option.

Your app is great, so I really hope someone will take the time to fix this. Thank you!

Regards,
Ari

**Comment by smurfy** [comment]
Hi Ari, 

i can reproduce the problem with Åros on Sailfish as-well. If i search Aros i find stations starting with Åros and if i search for Åros it looks like it ignores the char completely and searches for Ros instead.

I will have a look

**Comment by abkro** [comment]
Hi 

That's great! Thanks! 

Ari 

Den 14.04.2016 15:20, skrev Philipp Andreas: 

> Hi Ari, 
> 
> i can reproduce the problem with Åros on Sailfish as-well. If i search Aros i find stations starting with Åros and if i search for Åros it looks like it ignores the char completely and searches for Ros instead. 
> 
> I will have a look 
> 
> --
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub [1]
 

Links:
------
[1] https://github.com/smurfy/fahrplan/issues/189#issuecomment-209938068
